aten/src/ATen/NumericUtils.h

```diff
@@ -118,6 +136,14 @@ inline C10_HOST_DEVICE bool _isinf(at::Float8_e4m3fn val) {
   return false;
 }
 
+inline C10_HOST_DEVICE bool _isinf(at::Float8_e5m2fnuz val) {
+  return false;
```

Comment:
I dont' have enough context but this type doesn't have infinity therefore it doesn't make sense to thave `__isinf` defined. 

https://github.com/pytorch/pytorch/blob/b8ce05456cd6e332746165e469ab1e9048c3daf5/c10/util/Float8_e4m3fnuz-inl.h#L41

https://en.cppreference.com/w/cpp/numeric/math/isinf 

std::isinf is only defined floating points which have infs and not other types.

Reply:
`_isinf` and `_isnan` are defined for all types including integers, regardless of whether they have special values. This makes generic code easier to write.